ui/app/pages/create-account/connect-custodial/trustvault/components/star-placeholder/star-placeholder.component.js

```diff
@@ -0,0 +1,17 @@
+import React, { PureComponent } from 'react'
+import classnames from 'classnames'
+
+const CLASSNAME_STAR_PLACEHOLDER = 'star-placeholder'
+const CLASSNAME_INLINE_BLOCK = 'inline-block-child'
+
+export default class StarPlaceholder extends PureComponent {
+  render () {
+    return (
+      <span
+        className={classnames(CLASSNAME_STAR_PLACEHOLDER, CLASSNAME_INLINE_BLOCK)}
```

Comment:
These constants and this usage of classnames are all unnecessary, can we inline this `className` attr?